modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/fragments/partner-level-dashlet/index.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import React, {useEffect, useState} from 'react';
+
+import Container from '../../common/components/container';
+import AuthorizedPartnerIcon from '../../common/components/icons/AuthorizedPartnerIcon';
+import GoldPartnerIcon from '../../common/components/icons/GoldPartnerIcon';
+import PlatinumPartnerIcon from '../../common/components/icons/PlatinumPartnerIcon';
+import SilverPartnerIcon from '../../common/components/icons/SilverPartnerIcon';
+import LevelProgressBar from '../../common/components/level-progress-bar';
+import {ChartTypes} from '../../common/enums/chartTypes';
+import {PartnershipLevels} from '../../common/enums/partnershipLevels';
+import {partnerLevelData} from '../../common/mock/mock';
+import ClayIconProvider from '../../common/utils/ClayIconProvider';
+
+const PartnerIcon = ({level}) => {
+	if (level === PartnershipLevels.SILVER) {
+		return <SilverPartnerIcon />;
+	}
+	else if (level === PartnershipLevels.GOLD) {
+		return <GoldPartnerIcon />;
+	}
+	else if (level === PartnershipLevels.PLATINUM) {
+		return <PlatinumPartnerIcon />;
+	}
+
+	return <AuthorizedPartnerIcon />;
+};
+
+const CheckBoxItem = ({children, completed, text, title}) => {
+	const CheckIcon = () => {
+		if (completed) {
+			return (
+				<ClayIcon
+					className="m-0 text-brand-primary"
+					symbol="check-circle"
+				/>
+			);
+		}
+
+		return <ClayIcon className="m-0 text-danger" symbol="times-circle" />;
+	};
+
+	return (
+		<div className="d-flex mb-4">
+			<div
+				className={classNames('d-flex p-0 align-items-center', {
+					'col': !children,
+					'col-3': children,
+				})}
+			>
+				<ClayIconProvider>
+					<CheckIcon />
+				</ClayIconProvider>
```

Comment:
```suggestion
				<ClayIconProvider>
					<CheckIcon />
				</ClayIconProvider>
```

It's good to add the ClayIconProvider on main component